gravitee-node-license/src/main/java/io/gravitee/node/license/LicenseFeaturesServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.node.license;
+
+import static java.util.stream.Collectors.toSet;
+
+import io.gravitee.common.component.LifecycleComponent;
+import io.gravitee.common.service.AbstractService;
+import io.gravitee.node.api.Node;
+import io.gravitee.node.api.license.Feature;
+import io.gravitee.node.api.license.License;
+import io.gravitee.node.api.license.LicenseFeaturesService;
+import io.gravitee.node.api.license.LicenseModelService;
+import io.gravitee.node.api.license.model.LicenseModel;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Antoine CORDIER (antoine.cordier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Component
+public class LicenseFeaturesServiceImpl implements LicenseFeaturesService {
+
+    private static final String LICENSE_TIER_KEY = "tier";
+    private static final String LICENSE_PACKS_KEY = "packs";
+    private static final String LICENSE_FEATURES_KEY = "features";
+    private static final String LEGACY_FEATURE_VALUE = "included";
+    private static final String LIST_SEPARATOR = ",";
+
+    final Node node;
+
+    final LicenseModelService licenseModelService;
+
+    public LicenseFeaturesServiceImpl(Node node, LicenseModelService licenseModelService) {
+        this.node = node;
+        this.licenseModelService = licenseModelService;
+    }
+
+    @Override
+    public String getTier() {
+        return readString(LICENSE_TIER_KEY).orElse(null);
+    }
+
+    @Override
+    public Set<String> getPacks() {
+        return readList(LICENSE_PACKS_KEY);
+    }
+
+    @Override
+    public Set<String> getFeatures() {
+        HashSet<String> features = new HashSet<>();
+        LicenseModel licenseModel = licenseModelService.getLicenseModel();
+        features.addAll(readList(LICENSE_FEATURES_KEY));
```

Comment:
This should set when the licence is loaded, for instance a refresh method could be called and then all license model items are parsed once.